src/components/TimeQuerier/index.tsx

```diff
@@ -132,7 +132,14 @@ const TimeQuerier: React.FC = () => {
           </NumberInputStepper>
         </NumberInput>
       </Box>
-      <Box d="flex" pt="2" pb="1" w="100%">
+      <Box
+        d="flex"
+        alignItems="center"
```

Comment:
I think we are better off with a `textAlign: center` property. A flexbox with one child (Button) does not make sense to me